changelog.txt

```diff
@@ -3,9 +3,10 @@
  * Fixed a bug which might cause failure when loading the native library.
  * Fixed a bug which might trigger a timeout in Context.finalize().
  * Fixed a bug which might cause RealmObject.isValid() to throw an exception if the object is deleted.
- * Updated Realm core to version 0.89.7
-   - fixed a potential stack overflow issue which might cause a crash when encryption was used.
-   - embedded crypto functions from OpenSSL into Realm dynamic lib to avoid random issues on some devices.
+ * Updated Realm core to version 0.89.8
+   - Fixed a potential stack overflow issue which might cause a crash when encryption was used.
+   - Embedded crypto functions from OpenSSL into Realm dynamic lib to avoid random issues on some devices.
```

Comment:
I'm not against letting people know what we are using (in git log). But I find we should leave out implementation details in the change log.